src/js/dp/improve-this-page.js

```diff
@@ -9,6 +9,9 @@ document.addEventListener("DOMContentLoaded", function () {
     '<span id="feedback-form-error role="alert"">Something went wrong, try using our <a href="/feedback">feedback form</a>.</span>';
   let feedbackPositive = false
 
+  const useFeedbackAPI = document.querySelector("#feedback-api-enabled").value;
+  const feedbackAPIURL = document.querySelector("#feedback-api-url").value;
```

Comment:
These are chained properties, if JS can't find them the console will error; can you move the `.value` behind a guard clause?
i.e. where you call `feedbackAPIURL` on line 57; it could be `feedbackAPIURL.value`